docs/docs/snippets/how-to-create-dimensions.mdx

```diff
@@ -122,7 +132,7 @@ models:
     description: "A table of all orders."
     columns:
       - name: status
-        description: "Status from org256 settings codes. Referenced at 
+        description: "Status from org256 settings codes. Referenced at
         delivery from stat5 zone."
```

Comment:
hmm do we need these 2 lines on the same line?

Reply:
Seems like it's an existing issue, fixed!